presto-hive/src/main/java/com/facebook/presto/hive/ParquetHiveRecordCursor.java

```diff
@@ -808,6 +805,26 @@ public ParquetListConverter(Type prestoType, String columnName, GroupType listTy
             }
         }
 
+        //copied over from Apache Hive
+        private boolean isElementType(parquet.schema.Type repeatedType, String parentName)
+        {
+            if (repeatedType.isPrimitive() ||
+                    (repeatedType.asGroupType().getFieldCount() != 1)) {
+                return true;
+            }
+            else if (repeatedType.getName().equals("array")) {
```

Comment:
I think the preferred style in the code base is to not use `else if` if the previous `if` has a `return` in it